lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -647,7 +671,7 @@ def _create_course(self, *course_modes, **kwargs):
             modulestore().update_item(course, ModuleStoreEnum.UserID.test)
 
         for course_mode in course_modes:
-            min_price = (self.MIN_PRICE if course_mode != "honor" else 0)
+            min_price = (0 if course_mode in ["honor", "audit"] else self.MIN_PRICE)
```

Comment:
@rlucioni "Audit" mode is sort of deprecated :)  There are definitely still courses that use audit mode due to legal restrictions.